mccodec/mc/play/inventory/serverbound_use_item.mbt

```diff
@@ -0,0 +1,37 @@
+///|
+/// Use Item packet (0x40)
+/// Uses held item
+struct UseItemC2S {
+  hand : Int // Hand used enum(0=main, 1=off)
```

Comment:
use a bool and rename to `main_hand`. As mojang obviously not willing to add third hand... at least in 2025.